src/main/java/com/ecore/roles/web/rest/MembershipsRestController.java

```diff
@@ -30,25 +31,22 @@ public ResponseEntity<MembershipDto> assignRoleToMembership(
             @NotNull @Valid @RequestBody MembershipDto membershipDto) {
         Membership membership = membershipsService.assignRoleToMembership(membershipDto.toModel());
         return ResponseEntity
-                .status(200)
+                .status(201)
```

Comment:
Here the http status code response can be Created 201. Doesn't it make more sense?